scripts/runfabtests.sh

```diff
@@ -191,18 +239,19 @@ function unit_test {
 		return
 	fi
 
-	start_time=$(date '+%s.%N')
+	start_time=$(date '+%s')
 
-	${ssh} ${SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
+	${SSH_SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
 	p1=$!
 
 	wait $p1
 	ret1=$?
 
-	end_time=$(date '+%s.%N')
+	end_time=$(date '+%s')
 	test_time=$(compute_duration "$start_time" "$end_time")
 
-	if [ $ret1 -eq 61 ]; then
+        local nr=$(not_run $ret1)
```

Comment:
I think the fi_msg_epoll test should always be built, but on OSX (or wherever HAVE_EPOLL is not defined) msg_epoll.c should return -61 (notrun).

Or, does it need to be the linux specific epoll, why not just poll?

Either way hacking around it in the test automation is not the way to go.